database/Travel.js

```diff
@@ -1,12 +1,24 @@
-var Promise = require('bluebird');
-var sqlite3 = Promise.promisifyAll(require('sqlite3').verbose());
-var Travel = new sqlite3.Database('./travel.db');
+// var Promise = require('bluebird');
+// var sqlite3 = Promise.promisifyAll(require('sqlite3').verbose());
+// var Travel = new sqlite3.Database('./travel.db');
+var Travel = require('./index.js');
 
-var travelData = function() {
-  return Travel.all(`SELECT * FROM events WHERE id=1`, function(err, rows) {
-    if (err) console.error(err);
-    else return rows;
-  });
-}
+var getTravelData = function() {
+  var randomId = Math.round(Math.random() * 100);
+  console.log('getTravelData is invoked');
+  return Travel.getAsync(`SELECT * FROM events WHERE id=${randomId}`)
+    .then((rows) => {
+      console.log("Promise resolved");
+      console.log(rows);
 
-module.exports.travelData = travelData;
\ No newline at end of file
+      // if (err) {
+      //   console.error("error is:" + err);
+      // } else {
+      //   console.log('rows are:' + rows);
+      //   return rows;
+      // }
+    })
+    .catch((err) => console.error(err));
+};
+
+module.exports.getTravelData = getTravelData;
```

Comment:
Hey, So since you are only exporting this one function you can set it directly to exports.  That will alleviate you having to destructure it when you require it into a file.

Reply:
Thanks. I'll apply the changes now.